cmd/modelcmd/clientstore.go

```diff
@@ -27,6 +28,12 @@ func (s QualifyingClientStore) QualifiedModelName(controllerName, modelName stri
 		return "", nil
 	}
 	if !jujuclient.IsQualifiedModelName(modelName) {
+		// If the name is not a qualified name but the modelName is still "controller"
+		// we can assume that we actually want to access the controller
+		// return the fully qualified controller name if we try to access the controller model
+		if modelName == bootstrap.ControllerModelName {
```

Comment:
We might want to discuss whether we want this to be here.
Maybe that would be too invasive

Reply:
@mitechie ^ It all depends on whether we want this behavior in other places as well (switch/show-model/offer...).